H-Index-II.java

```diff
@@ -0,0 +1,28 @@
+// Time Complexity: O(N) best case, O(logN) worst case
```

Comment:
Please review complexity. its opposite.

Average case (log(n) and worst case o(n)